docs/services/postgresql/add-standby-node.md

```diff
@@ -32,6 +32,84 @@ $ cchq <env> ap setup_pg_standby.yml -e standby=[standby node]
 
 
 # Promoting a hot standby to Master
+1. Begin downtime for your site: `$ commcare-cloud <env> downtime start`
```

Comment:
nit: I would always try and put code into blocks, labeled with their language (here, `bash`) rather than inline for consistency.

Reply:
https://github.com/dimagi/commcare-cloud/pull/3072/commits/1ffddd10dbe0c096eee6295cff808e291b191966#diff-f05f8aa174fde4064a27d369a976954fR81